### PR TITLE
Add rule for FreeCallsCanada.co.uk

### PR DESCRIPTION
**Pull request by davecardwell**
Add rule for https://www.freecallscanada.co.uk/

### CODE REVIEW

File: src/chrome/content/rules/FreeCallsCanada.co.uk.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<ruleset name="FreeCallsCanada.co.uk">
+	<target host="freecallscanada.co.uk" />
+	<target host="www.freecallscanada.co.uk" />
+
+	<test url="http://freecallscanada.co.uk/" />
+	<test url="https://freecallscanada.co.uk/" />
+	<test url="http://www.freecallscanada.co.uk/" />
+
+	<rule from="^https?://freecallscanada\.co\.uk/" to="https://www.freecallscanada.co.uk/" />
```
Comment (J0WI) [inline src/chrome/content/rules/FreeCallsCanada.co.uk.xml]: You should not touch https domains and forcing to www as long as this works server side.

Comment (davecardwell) [inline src/chrome/content/rules/FreeCallsCanada.co.uk.xml]: @J0WI Thank you for checking. I have updated the regular expression and removed the redundant tests.

**Comment by J0WI** [comment]
Still no need to force the redirect to www, because this already works server-side.
Same on #4152.

Btw: If those hosts are related, you could merge the files or add a comment for each other.

**Comment by davecardwell** [comment]
@J0WI There are likely to be several more of these sites, so would the preferred method be to have a single “FreeCallsToX.xml” file that contained the rules for all of them?

**Comment by J0WI** [comment]
If one company owns all those services, use the Company name.
If there is really a bunch of related hosts and the rule becomes much more complex, group the hosts in whatever makes sense and give a hint in comments to the related files.

**Comment by davecardwell** [comment]
Rules moved to the existing localphone.com.xml in #4199

**Pull request closed without merging**